### DISCUSSION TITLE
Windows平台是否支持如何使用？

### DISCUSSION THREAD

**Discussion by Gwwwww777** [DualSubs/.github#5] [Q&A 问答] [answered]
支持列表中标注了有一些应用是支持在Windows平台上进行使用的，但是并没有针对Windows平台的使用教程。

**Comment by VirgilClyne** [comment, marked as answer]
任何终端设备
只要您在其上面安装相同的 `MitM` 用的`根证书`并信任
并使其网络流量通过 `Loon` `Surge` `Stash` 等的`网络代理`或`网关`
均可使用此项目
包括但不限于 `Windows` `Android` `Android TV` `LG WebOS` 等
